src/OutboundServer/api.yaml

```diff
@@ -581,6 +637,76 @@ components:
         transferRequestExtensions:
           $ref: '#/components/schemas/extensionList'
 
+    requestToPayInboundRequest:
```

Comment:
@ksatya77 Is this incoming to the DFSP or outgoing from the DFSP to switch?
If it is outgoing request sent by the DFSP using the scheme adapter I think the request ID should be generated in the SDK in same way as transferId is generated for outgoing transfers....the response to the backend should contain the ID so the DFSP backend can record it for later use if necessary.

Reply:
Updated the schema names to be requestToPayRequest and requestToPayResponse to avoid confusion and also to align with the naming convention used for /transfers.